### DISCUSSION TITLE
File versioning with Nextcloud AIO

### DISCUSSION THREAD

**Discussion by karlemilnikka** [nextcloud/all-in-one#1106] [Questions] [answered]
Is file versioning supposed to work with Nextcloud AIO? I’ve enabled Versions app (default) and the versions tab shows up on the details charm, but no versions are saved. This is regardless of whether I update the files using Nextclouds sync app for macOS or edit the files directly on the server using Nextcloud Office (Collabora Online). All changes are however indicated in the activity feed. 

Do I need to change something to get versioning working on Nextcloud AIO?

**Comment by szaimen** [comment]
I cannot reproduce this. Works for me:
![image](https://user-images.githubusercontent.com/42591237/188272573-0ef8273c-1529-468b-976c-d820b60346e5.png)

**Comment by karlemilnikka** [comment]
Thank you very much for taking your time to test it, @szaimen. I’ll investigate further on my end now that I know it is supposed to work.

**Comment by karlemilnikka** [comment, marked as answer]
I found what caused the issue on my end. There seems to be a conflict with the official end-to-end encryption app. Even though the files weren’t stored in an end-to-end encrypted folder, just having the app installed broke file versioning. After uninstalling the app, everything works as expected.

**Reply by karlemilnikka** [reply]
I was going to submit a ticket on the E2EE app’s page (even though it seems like the app is in pure maintenance mode and almost abandoned). Someone had already reported the issue back in 2021, so the issue is not related to Nextcloud AIO.
https://github.com/nextcloud/end_to_end_encryption/issues/247
